x/ibc-rate-limit/testutil/chain.go

```diff
@@ -76,12 +75,8 @@ func SignAndDeliver(
 	)
 
 	// Simulate a sending a transaction and committing a block
-	app.BeginBlock(abci.RequestBeginBlock{Header: header})
 	gInfo, res, err := app.Deliver(txCfg.TxEncoder(), tx)
 
-	app.EndBlock(abci.RequestEndBlock{})
-	app.Commit()
```

Comment:
LGTM, though I don't see any relation to rest of the PR

Reply:
This function overrides a function from the ibc testutils. They changed it so begin/end block must be called before and after, so if we keep this here all tests fail because of calling those twice